lib/stdlog.js

```diff
@@ -70,17 +70,54 @@
 
     // Use tail -F logs/generator_latest.txt to see the latest output
 
-    function writeToLog(data, encoding) {
+    function writeToLog(chunk, encoding, colorFunction) {
+        // We stopped logging in a previous call to write: return now
+        if (_logBytesWritten > LOG_SIZE_LIMIT) { return; }
```

Comment:
This block was largely hoisted out of the previous version of `logStream` so that it is common to the two new readable/writable flavors.